docs/maintainers/maintaining-k8s-compatibity.md

```diff
@@ -0,0 +1,31 @@
+# Maintaining Kubernetes Compatibility
+
+Upon the release of a Kubernetes minor release (e.g. [v1.28.0, on 15th August 2023](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.28)),
+the maintainer in charge of release should do the following to maintain
+compatibility with this K8s release.
+
+1. Check out its changelog to see if there are any changes that may affect its
+   compatibility with Antrea.
+
+2. Run Kubernetes Conformance tests with the new K8s minor release and the
+   Antrea releases under maintenance (the two most recent minor releases).
+   - For Linux, you can run the tests through [the workflow](https://github.com/antrea-io/antrea/actions/workflows/conformance.yml)
+     Click on `Run workflow`, enter the Antrea versions to test (e.g.
+     `v1.12.0`, `v1.11.2`), enter the K8s versions to test (e.g. `v1.28.0`).
+     You typically do not need to specify the Antrea Chart values and change
+     the test suite to run.
```

Comment:
Maybe:
```
     Typically, there is no need to specify the Antrea Chart values and change
     the test suite to run.
```
Or
```
    You typically can use the default Antrea Chart values and test suite to run.
```
One question, do we just need to run the default `whole-conformance` for the compatibility check?